src/prof_sys.c

```diff
@@ -264,6 +268,429 @@ prof_backtrace_impl(void **vec, unsigned *len, unsigned max_len) {
 	BT_FRAME(125)
 	BT_FRAME(126)
 	BT_FRAME(127)
+	BT_FRAME(128)
```

Comment:
Let's cut the static max for the PROF_GCC to 256. Also we should return an error code, if the user attempts to set a value greater than the static max.